rhel-minimal/Dockerfile

```diff
@@ -0,0 +1,76 @@
+ARG RHEL_VERSION=7
+
+FROM registry.access.redhat.com/ubi${RHEL_VERSION}/ubi-minimal
+
+ARG KONG_VERSION=2.8.1
+ENV KONG_VERSION $KONG_VERSION
+
+ARG KONG_SHA256="4f2d073122c97be80de301e6037d0913f15de1d8bb6eea2871542e9a4c164c72"
+
+LABEL name="Kong" \
+      vendor="Kong" \
+      version="$KONG_VERSION" \
+      release="1" \
+      url="https://konghq.com" \
+      summary="Next-Generation API Platform for Modern Architectures" \
+      description="Next-Generation API Platform for Modern Architectures"
+
+COPY LICENSE /licenses/
+
+ARG ASSET=ce
+ENV ASSET $ASSET
+
+ARG EE_PORTS
+
+COPY kong.rpm /tmp/kong.rpm
+
+ARG RHEL_VERSION
+ENV RHEL_VERSION $RHEL_VERSION
```

Comment:
Dockerfile.rpm gets the redhat version from `/etc/os-release` while this one is grabbing it from a Dockerfile arg.

I suspect there's `$reasons` just curious what they are

Reply:
I think the main reason is that in the past, we assumed the rhel "version" was a single digit, as with the `registry.access.redhat.com` ubi docker registries/repositories.

In reality, rhel-family versions have always been exclusively 3 plus digits: `7.9`, `8.6`, etc. This includes more recent additions to the family like the `redhat/ubi[89]` docker hub repos-- they don't actually offer a single-digit floating tag like `8`, only `8.6`. This also effects kbt. Feeding `RESTY_IMAGE_BASE=redhat/ubi8 RESTY_IMAGE_TAG=8.6` into it for example nets you a `kong-2.8.1.rhel8.6.amd64.rpm` package filename.